content/en/blog/_posts/2023-01-15-Vulnerable-Microservices/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2023-01-15
+slug: security-behavior
+---
+
+**Author:**
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy that relies on constructing a non-vulnerable, weakness-free service in 2022 is clearly too naïve. It seems the only viable strategy is to:
+
+|         |Admit that your services are vulnerable |
+|  ---    | ---  |
+
+In other words, consciously accept that you will lose any battle aimed to create non-vulnerable services. If your opponents find a single weakness as an entry-point, you lose. Admitting that although your best efforts, all your services are most likely vulnerable is an important first step. Next, we need to discuss what can you do about it...
+
+This post discuss the cyber challenges that Kubernetes users face when deploying microservices with Kubernetes and how to overcome them by observing the security-behavior of clients and services. It also points to [Guard](http://knative.dev/security-guard), an open source project offering security-behavior monitoring and control of your microservices deployed on a Kubernetes platform, given that they are presumed vulnerable.
```

Comment:
I think this should be something like:
```suggestion
This post discuss the cyber challenges that Kubernetes users face when deploying microservices with Kubernetes and how to overcome them by observing the _security relevant behavior_ of clients and services. It also points to [Guard](http://knative.dev/security-guard), an open source project offering security-behavior monitoring and control of your microservices deployed on a Kubernetes platform, given that they are presumed vulnerable.

What I mean by “security relevant behavior” is …
```
but, there's a problem. I couldn't guess what to put in place of the … because it's not obvious to me from the article what you mean by the term security-behavior. I'd prefer to fix that before publication, or at least to find a way to narrow the gap.

Reply:
_security relevant behavior_ does not sound right

Reply:
sent an alternative